src/platform-integration/web/wasm.md

```diff
@@ -68,22 +78,47 @@ At the bottom of the output, you should find experimental Wasm options like:
 
 ```console
 Experimental options
-    --wasm                                              Compile to WebAssembly rather than JavaScript.
-                                                        See https://flutter.dev/wasm for more information.
-    --omit-type-checks                                  Omit type checks in Wasm output.
-                                                        Reduces code size and improves performance, but might affect runtime correctness. Use with care.
-    --wasm-opt                                          Optimize output wasm using the Binaryen (https://github.com/WebAssembly/binaryen) tool.
-
-          [debug]                                       Similar to `full`, but member names are preserved. Debugging is easier, but size is a bit bigger.
-          [full] (default)                              wasm-opt is run. Build time is slower, but output is smaller and faster.
-          [none]                                        wasm-opt is not run. Fastest build; bigger, slower output.
+    --wasm                                                   Compile to WebAssembly rather than JavaScript.
+                                                             See https://flutter.dev/wasm for more information.
+    --[no-]strip-wasm                                        Whether to strip the resulting wasm file of static symbol names.
+                                                             (defaults to on)
 ```
 
 #### Pick a (simple) Flutter web application
 
-Choose a Flutter application without platform-specific packages or
-JavaScript interop code.
-These [known limitations](#known-limitations) cause issues with Wasm.
+Try the default template [sample app][], or choose any Flutter application
+that has been migrated to be
+[compatible with Wasm](#requires-js-interop-to-access-browser-and-js-apis).
+
+[sample app]: /get-started/test-drive
+
+#### Modify `index.html`
+
+Before building with Wasm, you'll need to modify the bootstrap logic in your
+`web/index.html` file.
+
+```html
+<!DOCTYPE HTML>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Web Benchmarks</title>
+  <script src="flutter.js"></script>
+</head>
+<body>
+  <script>
+    {{flutter_build_config}}
```

Comment:
UH OH! This is being treated by the renderer as...stuff.

We'll need to escape it!

Reply:
Oh yeah I figured out how to do this in my other draft PR. You can do

```
{% raw %}{{flutter_build_config}}{% endraw %}
```